hardware/arduino/avr/boards.txt

```diff
@@ -28,7 +28,7 @@ yun.upload.wait_for_upload_port=true
 yun.bootloader.tool=avrdude
 yun.bootloader.low_fuses=0xff
 yun.bootloader.high_fuses=0xd8
-yun.bootloader.extended_fuses=0xfb
+yun.bootloader.extended_fuses=0xcb
```

Comment:
Network flashing uses this workaround (https://github.com/arduino/openwrt-packages-yun/pull/1/files) so it always keeps the original fuses value when flashing.
Externally programming the Yun, instead, exposes the avrdude problem with masked bits, so for avrdude 6.3 some Yuns are not like other Yuns and you won't be able to program them.